controllers/volumesnapshotrestore_controller.go

```
@@ -88,15 +92,38 @@ func (r *VolumeSnapshotRestoreReconciler) Reconcile(ctx context.Context, req ctr
 		Name:      vsr.Name,
 	}
 
+	if VSRBatchNumber == 0 {
+		batchValue, err := GetRestoreBatchValue(vsr.Spec.ProtectedNamespace, r.Client)
+		if err != nil {
+			return ctrl.Result{}, err
+		}
+		VSRBatchNumber, _ = strconv.Atoi(batchValue)
```

Lets not ignore the error here as well.